auction-app-frontend/src/components/ProfileTab/ProfileTab.jsx

```diff
@@ -0,0 +1,110 @@
+import { useState } from "react";
+import { useForm } from "react-hook-form";
+
+import { Button, FormContainer, InputField } from "src/components";
+
+import { userProfilePicture } from "src/assets/images";
+import { dropdownInactive, dropdownActive } from "src/assets/icons";
+import { BUTTON_LABELS, BUTTON_VARIANTS } from "src/constants";
+import { personalInformationFormFields, cardInformationFields, addressInformationFields } from "src/forms/fields";
+
+import "./style.scss";
+
+const ProfileTab = () => {
+    const methods = useForm({
+        mode: "onBlur"
+    });
+
+    const [showCardInfo, setShowCardInfo] = useState(false);
+    const [showAddressInfo, setShowAddressInfo] = useState(false);
+
+    const toggleAdditionalInfo1 = () => setShowCardInfo(!showCardInfo);
+    const toggleAdditionalInfo2 = () => setShowAddressInfo(!showAddressInfo);
+
+    const onSubmit = (data) => {
+        console.log(data);
+    }
+
+    return (
+        <div className="profile-tab-container">
+            <FormContainer 
+                onSubmit={ onSubmit } 
+                methods={ methods } 
+                buttonLabel={ BUTTON_LABELS.SAVE_INFO }
+                buttonVariant={ BUTTON_VARIANTS.OUTLINED }
+            >
+                {/* general information */}
+                <div className="general-information">
+                    <div className="general-information-header">
+                        <span className="body-semibold">Personal Information</span>
```

Comment:
This can stay like this for now, because you aren't working on this right now, but consider in the future to extract this forms in components, because they look kinda same, so we can reduce unnecessary code.